### DISCUSSION TITLE
Refresh rate option for plugged and unplug

### DISCUSSION THREAD

**Discussion by meetlicious** [T-Troll/alienfx-tools#476] [Ideas]
Like new softwares from other company have a option to set the refresh rate based if the laptop is plugged or not .

Ex , if plugged set to 120hz and unpluged set to 48hz .

**Comment by T-Troll** [comment]
Monitor control is tricky in W10/W11, so i not sure it possible.

**Comment by meetlicious** [comment]
I think the Universal x86 Tuning Utility have this option for create a profile and set the refresh rate . Anyway i will be glad to se this comming to alienfx , Thanks for you work !!!
